### ISSUE TITLE
I'm interested to contribute on frontend

### ISSUE THREAD

**Issue by pradsdev** [Bookomate/bookomate#42]
Hey guys,

Sorry for creating an issue for this.

I'm interested in contributing on frontend side. I looked up on projects, couldn't find anything to work on. If this is still alive, please update some frontend related tasks on projects. I'll take it up.

**Comment by ck090** [comment]
Yup same here.. I would also love to build a better frontend and help in taking this project to it's goal of larger userbase

**Comment by jeremychrimes** [comment]
If you are interested in contributing feel free to download the code for the project and make modifications and then submit them as a pull request I will then review them and suggest modifications that need to be made before contributing them to the source code.
